ARMTemplates/Archimedes Capacity Scenario/readme.md

```diff
@@ -0,0 +1,217 @@
+
+![](Images/12.png)
+
+This will initiate the deployment. It should normally take about 5 minutes for the whole deployment to 
+complete.
+
+## Synapse pipeline template
+
+1.  After the pre-reqs are complete, navigate to the Synapse workspace just created
+
+![](Images/21.png)
+
+2.  Open the Synapse Studio
+
+![](Images/22.png)
+
+1.  Navigate to "Integrate -> Add new resource -> **Browse gallery**"
+
+![](Images/23.png)
+
+4.  Search for "SharePoint" and select the "Unlock Capacity Analytics and Insights for SharePoint and OneDrive using Microsoft 365 Datasets" template and Continue
+
+![](Images/24.png)
+
+5.  Create the new **Target** Linked services required by this pipeline
+
+![](Images/25.png)
+
+6.  Provide the parameters of the Linked Service
+    - Select Authentication Type = Service Principal 
+    - Use the storage account name, SPN id and secret (SPN key) from the pre-req steps above
+    - Test Connection and then click on Create
+
+![](Images/25a.png)
+
+
+7.  Repeat the Linked Service creation steps for the **source** Linked Service
+
+![](Images/25b.png)
+
+8.  Select "Open Pipeline"
+
+![](Images/26.png)
+
+
+9. Click on "Publish All" to validate and publish the pipeline
+
+![](Images/27.png)
+
+
+10. Review the changes and click Publish
+
+![](Images/27a.PNG)
+
+11. Verify that the pipeline has been successfully published
+
+![](Images/27b.PNG)
+
+12. Trigger the pipeline
+
+![](Images/27c.PNG)
+
+13. Provide the required parameters - StartTime, EndTime, StorageAccount, and StorageContainer created by the pre-req steps above (Note: names are case sensitive)
+
+![](Images/27d.PNG)
+
+14. You ran the pipeline created the request for the SharePoint Sites and SharePoint Files datasets which will be used to power the report! Once the admin consents to the request the data will be processed and delivered to your storage account.
+
+15.  You will see the data in the storage account. The report will pull the data from the uploaded capacity directory in your storage 
+
+![](Images/28.png)
+
+## PBI report template
+
+Note: In order to draw data to the Power BI report, you first need to run the Synapse pipeline as instructed in the previous section.
+
+The report will always pull the latest data from your Azure Data Lake Storage Gen 2. The report is to been as a template and is to be modified based on your requirements 
+
+Below steps will help to link datasets that are generated using Synapse pipeline above to link to Power BI 
+Template. 
+1. Download and install Microsoft Power BI Desktop if you don’t have it installed already on your machine. 
+    - Link to download Download Microsoft Power BI Desktop from Official Microsoft Download Center. [here](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
+2. Download the pre-created Power BI security report that can generate insights from data that is produced using Synapse pipeline in azure storage locations. Link to download Power BI Report. [here]()
```

Comment:
Replace "Link to download Power BI Report. [here]()" 
With "The link to download the Power BI Report will be available here when the template becomes available."